inc/deprecated/DeprecatedClassTrait.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace WP_Rocket\deprecated;
+
+/**
+ * Trait to use in a deprecated class, or a class containing deprecated methods.
+ */
+trait DeprecatedClassTrait {
+	/**
+	 * Marks a class as deprecated and informs when it has been used.
+	 * Similar to _deprecated_constructor(), but with different strings.
+	 * The current behavior is to trigger a user error if `WP_DEBUG` is true.
+	 *
+	 * @since  3.6
+	 * @author Grégory Viguier
```

Comment:
Remove all `@author` instances in this file.

Reply:
https://github.com/wp-media/wp-rocket/pull/2616/commits/dd041589183f9e6ea56fdbce81270869d5f90c2e